web/modules/custom/module1/module1.info.yml

```diff
@@ -0,0 +1,5 @@
+name: 'module1'
+type: module
+description: 'Custom module 1'
+package: Custom
+core_version_requirement: ^10 || ^11
```

Comment:
Why ^11?

Reply:
It was generated by drupal module builder.

Reply:
That is not a valid explanation. 
Any specific purpose for this ?

Reply:
No.
Removing ^11.